src/components/ProfileContent/ProfileContent.tsx

```diff
@@ -20,9 +21,19 @@ import { EditUserDataType } from '../../interfaces/User';
 import { toError } from '../../utils/error';
 import ProfileCard from '../ProfileCard/ProfileCard';
 
+type UpdateProfile = {
```

Comment:
Please, check this video: 
https://watch.screencastify.com/v/1mn0P8wVmlTWbJaqgP51
1. Country, job role, birthday are not showing on first load.
2. When profile image is changed and user wants to go to another page without saving it, confirm window not showing.
3. After that if I try to change something else and go to dashboard without saving, confirm window is not showing again.